app/design/adminhtml/default/default/template/widget/grid.phtml

```diff
@@ -114,7 +114,7 @@ $numColumns = count($this->getColumns());
                 <?php if ($this->getHeadersVisibility()): ?>
                     <tr class="headings">
                     <?php foreach ($this->getColumns() as $_column): ?>
-                        <th<?php echo $_column->getHeaderHtmlProperty() ?>><span class="nobr"><?php echo $_column->getHeaderHtml() ?></span></th>
```

Comment:
Not sure about removing the span element. It may break custom styling that select for it, for example:

```css
.grid tr.headings th span.nobr { ... }
```

Reply:
It is true (I thought about it as I'm very strict with other changes like modifying the signature of a public method) but as the same time the possibility of this happening and the minor side effect made me go for it.

Reply:
we could merge this in v21 but... it seems overkill for this.

Reply:
also, I've just removed the .grid tr.headings th span.nobr CSS line which was a hack for firefox 3... firefox 3...